src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -16,8 +16,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Emit
 {
     internal sealed class CSharpSymbolMatcher : SymbolMatcher
     {
-        private static readonly StringComparer s_nameComparer = StringComparer.Ordinal;
```

Comment:
Once I specified comparer explicitly in one case, I did it for other cases too. For consistency. 
Not all were showing in profiles.